lib/net/java/dev/spellcast/utilities/LockableListModel.java

```diff
@@ -679,14 +679,7 @@ public boolean retainAll( final Collection<?> c )
 		{
 			int originalSize = this.actualElements.size();
 
-			Iterator it = this.iterator();
-			while ( it.hasNext() )
-			{
-				if ( !c.contains( it.next() ) )
-				{
-					it.remove();
-				}
-			}
+          this.removeIf(o -> !c.contains(o));
```

Comment:
This looks space indented while the rest of the file is tab indented, probably because `lib` isn't affected by spotless.

Reply:
I just undid all of the `lib` changes